database/insertData/03_template_D_basicReviewTesting.js

```diff
@@ -269,6 +269,7 @@ exports.queries = [
                 level: 1
                 restrictions: { templateSectionRestrictions: ["S2"] }
               }
+              { permissionNameId: 10500, restrictions: { canSelfAssign: true }, stageNumber:1, level: 1 }
```

Comment:
Would you mind creating a new issue to migrate the `canSelfAssign` and `templateSectionRestrictions` to their own fields as we discussed, with a brief explanation about the "restrictions" field only being used for additional policy parameters.

Reply:
Created a placeholder note for this issue [here](https://github.com/orgs/openmsupply/projects/6#card-58400772)